core/schema/container.go

```diff
@@ -1288,31 +1288,31 @@ func (s *containerSchema) withoutFocus(ctx context.Context, parent *core.Contain
 	return child, nil
 }
 
-func (s *containerSchema) withDefaultShell(
+func (s *containerSchema) withDefaultTerminalCmd(
 	ctx context.Context,
 	ctr *core.Container,
 	args struct {
 		Args []string
 	},
 ) (*core.Container, error) {
 	ctr = ctr.Clone()
-	ctr.DefaultShell = args.Args
+	ctr.DefaultTerminalCmd = args.Args
 	return ctr, nil
 }
 
-func (s *containerSchema) shell(
+func (s *containerSchema) terminal(
 	ctx context.Context,
 	ctr dagql.Instance[*core.Container],
 	args struct {
-		Args dagql.Optional[dagql.ArrayInput[dagql.String]]
+		Cmd dagql.Optional[dagql.ArrayInput[dagql.String]]
```

Comment:
nit: this should be equivalent, if you want to clean up the boilerplate:

```suggestion
		Cmd *[]string
```